react/week4/github_user_search/src/githubApi.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useContext } from "react";
+import { UserSearchContext } from "./userSearch.js";
+
+const UserRepos = () => {
+  const userContext = useContext(UserSearchContext);
+  console.log(userContext);
+
+  const api = "https://api.github.com/users/HackYourFuture-CPH/repos";
+  console.log(api);
```

Comment:
remember to remove debugging commands :)

Reply:
:) ok kasper